mime_type.rb

```diff
@@ -17,16 +17,19 @@ def init_associations
     associations
   end
 
+  def match_file(filename)
+    splitted_fname = filename.split('.')
```

Comment:
Do not mix abbreviated variable names with full variable names. In Ruby, it's best to always use the full variable names.